account_filter_payable_receivable/README.rst

```diff
@@ -0,0 +1,44 @@
+account_filter_payable_receivable
```

Comment:
Don't put the technical name as title but the module name "Account Filter Payable Receivable"